### PR TITLE
Document the config for docker workspace cleanup

### LINKED ISSUE #16248: Document docker instance clean up configuration
## What

As we are re-adding process to scheduled jobs to clean up docker workspace, we should document how those job run:
* what do they remove and how to parameterize the behavior (which env vars)
* on which frequency

### PR DESCRIPTION
**Pull request by gosusnp**
## What

Document the config for docker workspace cleanup

Closes #16248

### CODE REVIEW

**Review by davinchia — APPROVED** [review]
:shipit:

**Review by colesnodgrass — APPROVED** [review]

File: docs/operator-guides/configuring-airbyte.md [hunk 1]
```diff
@@ -111,6 +111,11 @@ Set to empty values, e.g. "" to disable basic auth. **Be sure to change these va
 4. `MAX_DISCOVER_WORKERS` - Define the maximum number of Discover workers each Airbyte Worker container can support. Defaults to 5.
 5. `SENTRY_DSN` - Define the [DSN](https://docs.sentry.io/product/sentry-basics/dsn-explainer/) of necessary Sentry instance. Defaults to empty. Integration with Sentry is explained [here](./sentry-integration.md)
 
+#### Data Retention
+
+1. `TEMPORAL_HISTORY_RETENTION_IN_DAYS` - Define the retention period of the job history in Temporal, defaults to 30 days. When running in docker,
```
Comment (tryangul) [inline docs/operator-guides/configuring-airbyte.md:116]: Nit: Defines

**Review by tryangul — APPROVED** [review]

File: docs/operator-guides/configuring-airbyte.md (hunk 1 above)
Comment (gosusnp) [reply docs/operator-guides/configuring-airbyte.md:116]: Looking at the block above, I was shooting for consistency, do we want to fix all the `Define` to `Defines`?

Comment (gosusnp) [reply docs/operator-guides/configuring-airbyte.md:116]: discussed, leaving this as is.

**Pull request merged**